packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/types/write_batch.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class PlatformWriteBatch {
```

Comment:
docs. Also I think it would be clearer if this was named `PlatformWriteBatchHandle`

Reply:
Yes, docs are missing pretty much everywhere. I'm giving myself some time before I migrate this to add *Handle at the end, but I agree it's clearer.